vue-components/src/ImageDetection.vue

```diff
@@ -91,8 +93,20 @@ const annotationsWithColor = computed(() => {
 });
 
 const dpi = useDevicePixelRatio();
-const lineWidth = computed(() => LINE_WIDTH * dpi.pixelRatio.value);
 
+const { width } = useResizeObserver(visibleCanvas);
+
+const displayScale = computed(() => {
+  if (!visibleCanvas.value) return 1;
+  return imageSize.value.width / width.value;
+});
+
+const lineWidthInDisplay = computed(
+  () =>
+    props.lineWidth ?? LINE_WIDTH * dpi.pixelRatio.value * displayScale.value,
```

Comment:
not sure about it, should we scale the `props.lineWidth` as well?

Reply:
Yikes, yes we should.  Fixed.